### ISSUE TITLE
how to set big batch for transformer base model to accelerate training process

### ISSUE THREAD

**Issue by linzehui** [pytorch/fairseq#509]
Hi!
I have 8 V100 with 32G RAM , and I want to make the best use of the RAM to reproduce Transformer base model in WMT16 EN-DE dataset.So I think I should try to increase mini-batch. Could someone please give some examples or command lines  for big batch? It seems the paper Scaling Neural Machine Translation didn't give model configuration for transformer base model. And any tricks to make the training process as fast as possible  ? Thx!

**Comment by jiezhangGt** [comment]
Facing the same question

**Comment by linzehui** [comment]
@myleott 😊can you give some suggustions?

**Comment by frankang** [comment]
it's in the
> def add_dataset_args(parser, train=False, gen=False)
section in the options.py file

**Comment by huihuifan** [comment]
You can set the --update-freq to delay gradient updates until you have processed more samples, thus increasing the batch size, or you can try to set --max-tokens to a larger number to pack the GPU. The batching is already pretty efficient to reduce the amount of padding.

**Comment by linzehui** [comment]
@huihuifan But learning rate are closely correlated with batch. I am wondering what lr should I set if I set a larger batch

**Comment by frankang** [comment]
In some past issue posts the author gives some suggestions about LR and batchsize
